android/build.gradle

```diff
@@ -6,19 +6,19 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.0.1'
+        classpath 'com.android.tools.build:gradle:4.1.3'
     }
 }
 
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 29
-    buildToolsVersion '29.0.3'
+    compileSdkVersion 30
+    buildToolsVersion '30.0.3'
 
     defaultConfig {
-        minSdkVersion 16
```

Comment:
do we have data on how this will affect our customers? We don't want to block anyone from using our API, right?

Reply:
This is a knock-on effect from the networking library (OkHttp) we use in the native SDK. From the latest major version it only supports mindSdkVersion 21+. Ideally we want to support as many Android versions as possible, but equally version 21 dates back to 2014 so that's still pretty broad.

Reply:
that seems fair. do we make previous versions of the SDK available to customers?

Reply:
Yeah, the whole release history is still available so they can pick up an older SDK version if they're not ready to drop support for these Android versions just yet.